Camera2D.gd

```diff
@@ -0,0 +1,16 @@
+extends Camera2D
+
+
+const DEAD_ZONE: int = 140
+@onready var alpaca = $"../Alpaca"
+
+func _process(_delta) -> void:
+
+	self.position = alpaca.position
+	
+func _input(event: InputEvent) -> void:
+	if event is InputEventMouseMotion:
+		var target = -event.relative
+		self.position = alpaca.position + target.normalized() * (target.length() - DEAD_ZONE) 
```

Comment:
Add tween for smoother transition. Check [docs](https://docs.godotengine.org/en/stable/classes/class_tween.html)